src/app/features/ui/filter-search-button/filter-search-button.component.scss

```diff
@@ -0,0 +1,103 @@
+@use './src/styles/colors' as st;
```

Comment:
Czy nie lepiej by było może użyć `as colors` zamiast `as st` żeby było od razu wiadomo z jakiego to pliku?